Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -24,6 +24,7 @@ public class RestRequest: NSObject  {
     // Check if there exists a self-signed certificate and whether it's a secure connection
     private let isSecure: Bool
     private let isSelfSigned: Bool
+    private let certificateName: String
```

Comment:
Wouldn't this make more sense as an optional?  Also, better named `clientCertificateName`? Also, comment needed.